src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -163,46 +127,43 @@ namespace AirflowNetwork {
         OAF,     // Distribution system OA
         REL,     // Distribution system relief air
         SMF,     // Specified mass flow component
-        SVF,     // Specified volume flow component
-        Num
+        SVF      // Specified volume flow component
     };
 
     // EPlus component Type
-    enum class iEPlusComponentType : int
+    enum class ComponentType : int
     {
-        Invalid = 0,
-        SCN = 1, // Supply connection
-        RCN = 2, // Return connection
-        RHT = 3, // Reheat terminal
-        FAN = 4, // Fan
-        COI = 5, // Heating or cooling coil
-        HEX = 6, // Heat exchanger
-        RVD = 7, // Reheat VAV terminal damper
-        Num
+        Unknown = 0,
```

Comment:
Same question.

Reply:
See other longer comment, I'm trying to get all these to the point where they're not read in or ever possibly invalid.